Delete Site: vector figures

adambbecker opened the pull request:
Replaces the export site and start over png figures with svg versions.

##### Delete site export

![screen shot 2015-12-08 at 1 07 03 pm](https://cloud.githubusercontent.com/assets/1427136/11663916/c74a9fe0-9dac-11e5-8130-e02849db6cfd.png)

`settings/delete-site/*site-url*`

##### Start over

![screen shot 2015-12-08 at 1 07 20 pm](https://cloud.githubusercontent.com/assets/1427136/11663942/e108f242-9dac-11e5-90b4-f83720548ee4.png)

`settings/start-over/*site-url*`

---

/cc @rickybanister

rickybanister commented:
Thanks for updating those, looks much better. I'm not really familiar how to do it, but I believe there is some process or tool to remove all the metadata from the svgs that Sketch adds.

cc @alternatekev who might know how?

adambbecker commented:
Cool, thanks @alternatekev I just ran them through https://github.com/svg/svgo which cleaned those svg files up pretty well.

kellychoffman commented:
Nice. This might be a good opportunity to add some color to the gray and white landscape.

adambbecker commented:
@kellychoffman: Yeah that could be interesting to try out. I'll keep this PR pretty focused on just vectorizing the current images, but I'll definitely circle back on the color idea in the future.

mtias commented on public/images/delete-site/export-content.svg:
```diff
@@ -0,0 +1 @@
+<svg width="158" height="174" viewBox="0 0 158 174" xmlns="http://www.w3.org/2000/svg"><title>figure</title><g transform="translate(0 -5)" fill="none" fill-rule="evenodd"><rect fill="#D5E5EB" x="57" y="157" width="45" height="22" rx="4"/><text font-family="Open Sans" font-size="11" font-weight="526" fill="#FFF"><tspan x="69.736" y="172">.ZIP</tspan></text><path d="M89.5 131.5h-6v-9h-9v9h-6L79 142 ...
```
may be better to inline these in the component?

adambbecker replied:
@mtias: I suppose we could, although I'm not sure necessarily any big benefit of doing so. Having that image tag makes the component a bit more readable to me, but we still could definitely put it inline.

mtias replied:
We'd avoid a request, and I think we may need whitelisting for non image files at the server level.

adambbecker replied:
Ah, cool, that's a good point. I'll switch them out.

mtias replied:
If we think we may be reusing these we may also look into making it simple components like https://github.com/Automattic/wp-calypso/blob/master/client/components/wordpress-logo/index.jsx

And with React 0.14 just pure functional components :)

adambbecker replied:
Yeah I see what you mean and we could definitely do that in the future. I'd rather do it however, at the point we actually find a concrete need for it. Trying to keep this PR as focused as possible.

mtias replied:
Absolutely :)

adambbecker commented:
@mtias: Cool, inline'd those svgs, removed the outdated images from `public`, rebased, and squashed.

Although after the rebase, CircleCi is now saying:

```
Happiness engineers actions
    1) "before each" hook for "Actions should be an object"

  Happiness engineers Store
    ✓ Store should be an object
    ✓ Store should have method getHappinessEngineers
    Get Happiness Engineers
      ✓ Should return empty array when there are no happiness engineers
      ✓ Should return an array of happiness engineers when there are happiness engineers


  4 passing (2s)
  1 failing

  1) Happiness engineers actions "before each" hook for "Actions should be an object":
     Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.

...

These tests have failed:
/home/ubuntu/wp-calypso/client/lib/happiness-engineers
```

Time to go track whatever that stuff is down.

adambbecker commented:
Alright, rebased again and that seemed to get pass the test.

rralian commented:
Looks good! :shipit:

mtias commented:
:+1:
